### PR TITLE
CU-cewwu4 Remove Deprecated `Function1<A, B>` data type.

### PR DESCRIPTION
**Pull request by raulraja**

### CODE REVIEW

**Review by i-walker — APPROVED** [review]

File: arrow-docs/docs/arrow/typeclasses/profunctor/README.md [hunk 1]
```diff
@@ -39,20 +39,6 @@ Functions are a binary type constructor of an input type and an output type. It
```
Comment (aballano) [inline arrow-docs/docs/arrow/typeclasses/profunctor/README.md:39]: there's a ref right above this line as well

File: arrow-docs/docs/arrow/typeclasses/profunctor/README.md [hunk 2]
```diff
@@ -61,18 +47,6 @@ Contramap on the first type parameter and map on the second type parameter.
 
 `fun Kind2<F, A, B>.dimap(fl: (C) -> A, fr: (B) -> D): Kind2<F, C, D>`
 
-```kotlin:ank
```
Comment (aballano) [inline arrow-docs/docs/arrow/typeclasses/profunctor/README.md:64]: Here and above we would need some other examples

Comment (nomisRev) [reply arrow-docs/docs/arrow/typeclasses/profunctor/README.md:64]: This typeclass will be removed in `0.13.0` and is deprecated in next version. I don't think we should invest more time on something that is planned to be removed soon.

**Comment by nomisRev** [comment]
https://github.com/arrow-kt/arrow-incubator/pull/147

**Review by nomisRev — APPROVED** [review]

**Review by franciscodr — APPROVED** [review]

**Review by aballano — APPROVED** [review]

**Pull request merged**